feature-utils/poly-look/src/css/radius.css

```diff
@@ -0,0 +1,12 @@
+:root {
```

Comment:
Since these are part of the design system, wouldn't it be useful to have a way of distinguishing them? Their own directory, or `pl-` as a prefix, something like that?

Reply:
It is true that these are all design tokens `foundations` or `global`. 
Nonetheless, when we we will create our tokens for more specific component, that will be done at the component folder level, so leaving this in a general folder is also ok I think

Reply:
What about then changing their name? Use `pl-` as a prefix.